packages/airnode-node/test/setup/e2e/testing.ts

```diff
@@ -5,8 +5,6 @@ import { buildChainConfig, buildProvider, getDeployerIndex } from './utils';
 import { deployAirnodeRrp, makeRequests } from './deployment';
 import { buildConfig, operation } from '../../fixtures';
 
-export const increaseTestTimeout = (timeoutMs = 120_000) => jest.setTimeout(timeoutMs);
```

Comment:
Don't we want to keep the function and use it at the top level? Since there's the same timeout everywhere in the e2e tests.